examples/ec2/v1beta1/fleetrequest.yaml

```
@@ -0,0 +1,75 @@
+# SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: Fleet
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ec2/v1beta1/fleetrequest
+  labels:
+    testing.upbound.io/example-name: xplane_se
+  name: xplane-se
+spec:
+  forProvider:
+    launchTemplateConfig:
+    - launchTemplateSpecification:
+        launchTemplateIdSelector:
+          matchLabels:
+            testing.upbound.io/example-name: example
+        versionSelector:
+          matchLabels:
+            testing.upbound.io/example-name: example
```

```suggestion
        version: 1
```
The selector for the version doesn't seem to be working properly, could you try the one above to make sure?

```
    logger.go:42: 12:39:56 | case/0-apply |       message: 'cannot resolve references: mg.Spec.ForProvider.LaunchTemplateConfig[i3].LaunchTemplateSpecification.Version:
    logger.go:42: 12:39:56 | case/0-apply |         referenced field was empty (referenced resource may not yet be ready)'
    logger.go:42: 12:39:56 | case/0-apply |       reason: ReconcileError
    logger.go:42: 12:39:56 | case/0-apply |       status: "False"
    logger.go:42: 12:39:56 | case/0-apply |       type: Synced
```